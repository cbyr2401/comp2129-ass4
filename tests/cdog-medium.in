0.85
7
A
B
C
D
E
F
G
7
A B
A C
A D
A E
A F
B G
C G

