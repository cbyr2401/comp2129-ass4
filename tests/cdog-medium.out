A 0.10167837
B 0.11889378
C 0.11889378
D 0.11889378
E 0.11889378
F 0.11889378
G 0.30385271
